resources/Schema/Components/Dash.xsd

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:t="types">
+	<xs:import schemaLocation="../Types.xsd" namespace="types"/>
+
+	<xs:element name="Dash">
```

Comment:
Rename to DashAbility or something similar so we can group it with other abilities more easily.